src/k-provider/ovp/response-types/kaltura-media-entry.ts

```diff
@@ -185,5 +191,6 @@ export class KalturaMediaEntry {
     this.plays = entry.plays;
     this.views = entry.views;
     this.downloadUrl = entry.downloadUrl;
+    this.rawThumbnailUrl = entry.thumbnailUrl;
```

Comment:
maybe put it next to poster initialization, in line 183

Reply:
done